tests/serial/time/explicit/serial-coupling/ReadWriteScalarDataFirstParticipantInitData.cpp

```diff
@@ -0,0 +1,80 @@
+#ifndef PRECICE_NO_MPI
+
+#include "testing/Testing.hpp"
+
+#include <precice/SolverInterface.hpp>
+#include <vector>
+
+using namespace precice;
+
+BOOST_AUTO_TEST_SUITE(Integration)
+BOOST_AUTO_TEST_SUITE(Serial)
+BOOST_AUTO_TEST_SUITE(Time)
```

Comment:
I don't think that this test (maybe also the other new test) fits very well into `Integration/Serial/Time`. Maybe `Integration/Serial/InitializeData` ? Or should we create a dedicated test suite for the `participant-first` configuration?

Reply:
In a way, that's an inherent problem / feature of integration tests; they always test multiple components simultaneously. Here it is "first-participant" + "data initialization". For me, "first-participant" is the dominating feature here. That's why I added it to `time`. Is somehow fits nicely as it only applies to serial coupling.